stdlib/REPL/src/REPL.jl

```diff
@@ -116,6 +117,9 @@ import Base:
     AnyDict,
     ==
 
+show(io::IO, ::MIME"text/plain", ex::Expr) =
```

Comment:
Won't this ignore the IOContext flags in `io`?

Reply:
Not any more :)